x/mint/module/abci.go

```diff
@@ -88,6 +89,7 @@ func BeginBlocker(ctx context.Context, k keeper.Keeper) error {
 		if err != nil {
 			return err
 		}
+		types.EmitNewEcosystemTokenMintSetEvent(sdkCtx, blockHeight, tokensToMint)
```

Comment:
I don't love how this is called only once per month but i see how it can get expensive to do it more than per block just for an event...

Reply:
@relyt29 fortunately also made `EmissionInfo` in `x/mint` query server so actually we can add that to our allora producer. Should be fine. And now this way we can have both